polaris-migrator/src/migrations/scss-replace-media-query-interpolation/transform.ts

```diff
@@ -0,0 +1,47 @@
+import type {API, FileInfo, Options} from 'jscodeshift';
+import type {Plugin} from 'postcss';
+import postcss from 'postcss';
+import stylelint from 'stylelint';
+
+const plugin = (): Plugin => {
+  return {
+    postcssPlugin: 'scss-replace-media-query-interpolation',
+    AtRule(atRule) {
+      if (atRule.name === 'media' || atRule.name === 'container') {
```

Comment:
Didn't dive too deep into the API, but wondering if there's a 'proper' way to do this.